style.css

```diff
@@ -254,3 +256,172 @@ textarea:hover {
 input:hover {
   border: 1px solid #253858;
 }
+
+@media screen and (min-width: 768px) {
+  .main {
+    background: url(images/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-color: #fff;
+    border-bottom-left-radius: 100px;
+  }
+
+  body {
+    background-color: #e5e5e5;
+  }
+
+  nav {
+    display: flex;
+    justify-content: space-between;
+    padding: 40px 190px 40px 90px;
+    background-color: #fff;
+  }
+
+  .nav-links {
+    display: flex;
+  }
+
+  .nav-links ul {
+    display: flex;
+    list-style: none;
+    gap: 35px;
+  }
+
+  .nav-links ul li a {
+    color: #344563;
+    text-decoration: none;
+    font-size: 15px;
+  }
+
+  .logo {
+    font-size: 18px;
+  }
+
+  h1 {
+    font-size: 45px;
+  }
+
+  .fa-bars {
+    visibility: hidden;
+  }
+
+  .presentation {
+    font-size: 20px;
+    padding-left: 0;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .lang {
+    padding-top: 0;
+    padding-bottom: 0;
+  }
+
+  .connect {
+    font-size: 16px;
+    padding-left: 0;
+  }
+
+  .icons ul li a {
+    font-size: 20px;
+  }
+
+  .icons {
+    margin-left: 0;
+    padding: 12px;
+  }
+
+  .content {
+    width: 85%;
+  }
+
+  .works {
+    background-color: #fff;
+    display: flex;
+    width: 70%;
+    margin-bottom: 100px;
+    margin-top: 100px;
+    height: 410px;
+  }
+
+  .part {
+    height: 350px;
+  }
+
+  #section2,
+  #section4 {
+    flex-direction: row-reverse;
+  }
+
+  .body {
+    margin-left: 250px;
+    width: 100%;
+  }
+
+  .photo {
+    width: 380px;
+    height: 370px;
+  }
+
+  .contentAndLangs {
+    background-color: #fff;
+    border-top-right-radius: 90px;
+    display: flex;
+    height: 900px;
+  }
+
+  #content2 {
+    width: 590px;
+  }
+
+  .langs {
+    width: 700px;
+    height: 170px;
+    margin-top: -5px;
+    margin-left: 10px;
+    padding: 80px 70px;
+  }
+
+  .js {
+    display: inline-flex;
+    flex-direction: column;
+    width: 100px;
+    flex-wrap: nowrap;
+    margin: 3px;
+  }
+
+  .contact {
+    background-image: url(images/contact\ form\ background\ shapes.svg);
+    background-repeat: no-repeat;
+    padding: 140px 120px;
+    background-color: #6070ff;
+    border-top-left-radius: 120px;
+    display: grid;
+    padding-left: 350px;
+    margin-top: -110px;
+  }
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 correct the position of the contact section content, it is centered in the Figma design. 👇🏽 

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/236584211-5ff372f6-9b7f-4cd4-b29d-f01b58fb22f5.png) | ![image](https://user-images.githubusercontent.com/58642949/236584241-06405bf9-c36d-4c63-90f7-978b4d388cf0.png) |